public/assets/icons/misc/add-outline.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512"><title>Add</title><path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32" d="M256 112v288M400 256H112"/></svg>
```

Comment:
Can we use https://ionicons.com instead of this custom icon?

Reply:
Suggested change implemented - was able to run it with your suggestion below.